go/vt/topo/etcd2topo/server.go

```diff
@@ -34,13 +34,21 @@ We follow these conventions within this package:
 package etcd2topo
 
 import (
+	"flag"
 	"strings"
 	"time"
 
 	"github.com/coreos/etcd/clientv3"
+	"github.com/coreos/etcd/pkg/transport"
 	"vitess.io/vitess/go/vt/topo"
 )
 
+var (
+	clientCertPath = flag.String("topo_etcd2_tls_cert", "", "the client cert to use to connect to the etcd topo server, requires topo_etcd2_tls_key, enables TLS")
+	clientKeyPath  = flag.String("topo_etcd2_tls_key", "", "the client key to use to connect to the etcd topo server, enables TLS")
```

Comment:
Add "path to" in front of each of these, to make it clear that we expect file paths, not literal contents.

Reply:
Done.